src/count_loc.jl

```diff
@@ -69,5 +83,184 @@ end
 count_readme(table) = count_docs(table, ("readme", "readme.md"))
 
 count_julia_loc(pkg::Package, args...) = count_julia_loc(pkg.lines_of_code, args...)
+count_docstrings(pkg::Package, args...) = count_docstrings(pkg.lines_of_code, args...)
 count_docs(pkg::Package, args...) = count_docs(pkg.lines_of_code, args...)
 count_readme(pkg::Package, args...) = count_readme(pkg.lines_of_code, args...)
+
+#####
+##### Custom Julia line counting
+#####
+
+# We can't do this at the level of SyntaxNode's because we've lost whitespace & comments already
+# So we will go to GreenNode's
+
+# Avoid piracy by defining AbstractTrees definitions on a wrapper
```

Comment:
Yea hmm. It might be nice to make `JuliaSyntax` depend on `AbstractTrees`. Though dependencies do make vendoring JuliaSyntax into Base a more difficult - we'd also have to vendor any dependencies.

Reply:
Personally I'd do the recursion pruning as part of the traversal (unless, conceptually, you want the wrapped tree to not contain children for these elements). But yea if it works for you then cool :+1:

Reply:
Seemed easier to rely on AbstractTrees traversal methods which I don’t think allow me to customize that beyond defining `children`